src/core/defs.jl

```diff
@@ -353,9 +353,9 @@ end
 """
     parameter_dimensions(obj::AbstractComponentDef, param_name::Symbol)
 
-Returns the names of the dimensions of parameter `param_name`.
+Return the names of the dimensions of parameter `param_name` exposed in the composite 
+component definition indicated by`obj`.
```

Comment:
Is `obj` always/only composite definitions? Or can it also just be a component definition?

Reply:
I think it can probably be either